src/routes/tutorial/[slug]/filetree/Item.svelte

```diff
@@ -188,24 +188,7 @@
 	}
 
 	[aria-current='true'] {
-		color: var(--prime);
+		color: hsl(15, 100%, 50%);
```

Comment:
Looks like the root issue here is that the `--prime` variable no longer exists - I think we should use `--sk-theme-1` instead of a hard-coded value

Reply:
I've change color to `--sk-theme-1`.